client/src/main/java/io/pravega/client/admin/impl/StreamManagerImpl.java

```diff
@@ -48,10 +48,16 @@
     private final ConnectionPool connectionPool;
     private final ScheduledExecutorService executor;
     private final StreamCutHelper streamCutHelper;
-    
+
+
     public StreamManagerImpl(ClientConfig clientConfig) {
+        this(clientConfig, ControllerImplConfig.builder().clientConfig(clientConfig).build());
```

Comment:
There are no unit tests that cover this constructor. I had this problem before and it is unfeasible to try to add any meaningful unit tests for this because it creates a real `ControllerImpl` that tries to connect to a real Controller. All the client unit tests use the other constructor that takes in a `Controller` object (which is a mock).

Unfortunately I had to touch this constructor because I needed a way to pass in the controller configuration. A simple review the code can show that I made no functional changes to the existing constructor.

Reply:
Ok, I see that this is being tested at least in a standalone integration test. Should we create an issue to see if we can refactor this class to make the constructors testable?

Reply:
I made #5637 to track this.